README.md

```diff
@@ -108,27 +155,27 @@ NimbleTemplate supports 4 variants:
 
 #### 2.2/ Phoenix project
 
-The Phoenix project could be either a Web or API project.
+The Phoenix project could be one of **Web**, **API**, or **Live View**.
 
-- Web variant supports HTML and Webpack configuration.
+- *Web variant* supports HTML and Webpack configuration.
 
 ```bash
 mix phx.new awesome_project
 ```
 
-- LiveView project is including HTML and Webpack configuration.
+- *LiveView variant* supports for LiveView assets and configuration. 
 
 ```bash
 mix phx.new awesome_project --live
 ```
 
-- API variant does NOT support HTML and Webpack configuration.
+- *API variant* does NOT generate HTML view and generate Webpack files.
```

Comment:
Should we remove the second `generate` before Webpack @byhbt  `does NOT generate HTML view and Webpack configuration`

Reply:
Updated!